File: 2016/A.java
package a2016;

import java.util.Scanner;

public class A {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int credito1 = sc.nextInt();
		int credito2 = sc.nextInt();
		int credito3 = sc.nextInt();
		
		if(credito1 == credito2 || credito1 == credito3 || credito2 == credito3) {
			System.out.println("S");
		}else if(
				(credito1+credito2)==credito3 || 
				(credito3+credito1)==credito2 || 
				(credito2+credito3)==credito1) {
			System.out.println("S");
		}else {
			System.out.println("N");
		}
		
	}

}

File: 2016/A.py
credito1 = int(input())
credito2 = int(input())
credito3 = int(input())

if credito1 == credito2 or credito1 == credito3 or credito2 == credito3:
    print("S")
elif (credito1+credito2) == credito3 or (credito3+credito1) == credito2 or (credito2+credito3) == credito1:
    print("S")
else:
    print("N")
